vector/v.in.pdal/v.in.pdal.html

```diff
@@ -1,15 +1,8 @@
 <h2>DESCRIPTION</h2>
 
-<em>v.in.pdal</em> supports the following PDAL filters:
+<p>Imports point cloud (e.g. from a LAS file) as vector points with PDAL library.
```

Comment:
Imports **a** point cloud (e.g. from a LAS file) as vector points with **the** PDAL library.